Application/Dtos/Shared/BasePaginationParameters.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Application.Dtos.Shared
+{
+    public class BasePaginationParameters
```

Comment:
Dtos Are Used To Save The Input You Get From Forms or Query Links And Do Operations On Them Instead Of Working With The Main Database Models Or Writing A List Of Query Options In The Endpoint Arugments